src/util/tracking.js

```diff
@@ -123,7 +123,7 @@ export const trackApi = log => (response, queryResponses, metadata={}) => {
 	}
 	// special case - login requests need to be tracked
 	const loginResponse = queryResponses.find(r => r.login);
-	if (loginResponse) {
+	if ((loginResponse && loginResponse.login.value || {}).member) {
```

Comment:
no - the `.find(r => r.login)` will only return something that has a truthy `.login` property